targets/TARGET_Freescale/TARGET_KSDK2_MCUS/TARGET_MCU_K64F/device/TOOLCHAIN_GCC_ARM/MK64FN1M0xxx12.ld

```diff
@@ -52,9 +52,11 @@ ENTRY(Reset_Handler)
 
 __ram_vector_table__ = 1;
 
-/* Heap 1/4 of ram and stack 1/8 */
-__stack_size__ = 0x8000;
-__heap_size__ = 0x10000;
+/* Heap 1/4 of ram and stack 1/8. 0x1000 is reduced from both heap
+ * and stack to accomodate static data introduced by GCOV.
+ */
+__stack_size__ = 0x7000;
+__heap_size__ = 0x09000;
```

Comment:
I'm not sure what you mean by "the changes do no comply with build profile", could you please elaborate? About Fast models, that's certainly a very good approach, but it should still be possible to run code coverage directly on the devices that have enough resources.